pyEpiabm/pyEpiabm/tests/test_unit/test_intervention/test_social_distancing.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+from unittest.mock import patch
+
+import pyEpiabm as pe
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.intervention import SocialDistancing
+from pyEpiabm.tests.test_unit.parameter_config_tests import TestPyEpiabm
+
+
+class TestSocialDistancing(TestPyEpiabm):
+    """Test the 'SocialDistancing' class.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Intialise a population with one infector and one
+        infectee, both in the same place and household.
+        """
+        super(TestSocialDistancing, cls).setUpClass()  # Sets up parameters
+        cls.time = 1
+        cls._population = pe.Population()
+        cls._population.add_cells(1)
+        cls._population.cells[0].add_microcells(1)
+        cls.microcell = cls._population.cells[0].microcells[0]
+        cls.microcell.add_people(1)
+        cls.person = cls.microcell.persons[0]
+        cls.person.update_status(InfectionStatus(7))
+
```

Comment:
We define a population a bit different in 'test_household_quarantine.py' and I think that way is more clear. Bit confused here because you are only adding one person with a certain InfectionStatus. Is there a second which is susceptible? 

I think this will be more clear:
```
cls.pop_factory = pe.routine.ToyPopulationFactory()
cls.pop_params = {"population_size": 2, "cell_number": 1,
                          "microcell_number": 1, "household_number": 1}
cls.test_population = cls.pop_factory.make_pop(cls.pop_params)
cls.sympt_person = cls.test_population.cells[0].microcells[
            0].persons[0]
cls.sympt_person.update_status(InfectionStatus.InfectMild)
cls.susc_person = cls.test_population.cells[0].microcells[0].persons[1]
cls.susc_person.update_status(InfectionStatus.Susceptible)
```

Reply:
Done! I also done the similar thing in case isolation and place closure tests.

Reply:
Great suggestion @HenrietteCapel !